pkg/cloudprovider/cloud.go

```diff
@@ -37,6 +37,8 @@ type Interface interface {
 	Clusters() (Clusters, bool)
 	// Routes returns a routes interface along with whether the interface is supported.
 	Routes() (Routes, bool)
+	// Volumes returns a volumes interface along with whether the interface is supported.
+	Volumes() (Volumes, bool)
```

Comment:
Does GCE/AWS/OpenStack support more than one?

It looks like each of their APIs has 1 kind of volume with many performance attributes that can be requested, but still just 1 kind in the API.